client/.env

```diff
@@ -1 +1,3 @@
+REACT_APP_AIRTABLE_API_KEY="keyczjtMi7gM7IUKE"
```

Comment:
Did you mean to include these into your PR? It's weird that these got past your .gitignore

Reply:
Thats so odd! no not sure why they went through!!

Reply:
We don't use these anymore. You should update your local to be the same as the remote master branch.

Reply:
They should be removed now.